hibernate-osgi/src/main/java/org/hibernate/osgi/OsgiClassLoader.java

```diff
@@ -212,4 +214,18 @@ public void clear() {
 		resourceCache.clear();
 	}
 
+	public void removeBundle(Bundle bundle) {
```

Comment:
@gnodet, good question.  *All* bundles should be solely added through `addBundle`, so the `Map` should be safe.  I'll take care of that later on.  Thanks!